src/net/sourceforge/kolmafia/request/ChatRequest.java

```diff
@@ -21,7 +21,9 @@ public ChatRequest(final long lastSeen, final boolean tabbedChat, final boolean
     StringBuilder newURLString = new StringBuilder("newchatmessages.php?");
 
     if (tabbedChat) {
-      newURLString.append("j=1&");
+      newURLString.append("aa=");
+      newURLString.append(Math.random());
```

Comment:
Why?

Reply:
It's both in the description and the commit message:

> KoL mchat is adding a random aa=%RANDOMNUMBER% parameter to each newchatmessages.php request. Probably some kind of cache buster? I've added this to ChatRequest for the mchat relay requests.

Reply:
OK.  Not everyone who is reading code is also reading description and commit messages and this was not something I expected from the description.  

Resolved.